### ISSUE TITLE
[Eclipse-ThirdParty][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Name property is empty for edit in "Install dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1369]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Help menu.
3. Navigate and select Install new software Menu item under Help menu.
4. Install dialog will open.
5. In "Install" dialog , tab to details edit and observe the name property using inspect.

**Actual Result:**
Name property is empty for the edit box

**Narrator behaviour:**
read-only text, Provides support for developing projects that help build, test and deploy Java applications for the Microsoft Azure cloud in Eclipse.

**NVDA behaviour:**
Details  grouping
edit  read only  multi line
Provides support for developing projects that help build, test and deploy Java applications for the  Microsoft Azure cloud in Eclipse.

**Expected Result:**
Name property should be simple and meaningful.
Name property for the edit should be "Details".

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![details edit-name-null](https://user-images.githubusercontent.com/22956130/35680313-fd8b07cc-077f-11e8-90a8-721b0c8ac0b3.jpg)
![details edit-name-nvda](https://user-images.githubusercontent.com/22956130/35680318-00c58656-0780-11e8-8054-7a16c1af8544.jpg)

**Comment by Eskibear** [comment]
It is determined by Eclipse IDE. We cannot fix it.

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533979

**Comment by Eskibear** [comment]
linked to #1619

**Comment by Eskibear** [comment]
* **Importance of experience**: somewhat important. It's a one-time installation.
* **Quality of experience**: not impacted. A simple workaround exists. There is another way to install the toolkit: Help -> Eclipse Marketplace... -> search for "azure toolkit" -> Install.
![market](https://user-images.githubusercontent.com/2351748/44134673-7f0f2560-a098-11e8-9b03-2036cd05b532.gif)

* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by WiproPSAccTest** [comment]
Agreed with the resolution . Can you please remove A11YExclusion, wontfix, Resolved, notreproduce labels wherever applicable and add ExclusionAgreed

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749821
